client/client-multi/client-objective-c/Kaa/KaaClientFactory.h

```diff
@@ -0,0 +1,44 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
```

Comment:
This file was actually renamed, not created. It's marked as new because new header with the same name was created.